src/tablo/Tablo.d.ts

```diff
@@ -76,8 +76,8 @@ export type TabloProps<Data, Columns> = {
 
 export const Header: React.StatelessComponent<HeaderProps>;
 export const Footer: React.StatelessComponent<FooterProps>;
-export const Cell: React.StatelessComponent<CellProps>;
+export const Cell: React.StatelessComponent<CellProps<any>>;
 export const ColumnGroup: React.StatelessComponent<ColumnGroupProps>;
-export const Column: React.StatelessComponent<ColumnProps>;
-declare const Tablo: React.ComponentClass<TabloProps>;
+export const Column: React.StatelessComponent<ColumnProps<string>>
+declare const Tablo: React.ComponentClass<TabloProps<any, string>>;
```

Comment:
because it should be an enum: `'column1' | 'column2' | 'column3' ...`
I pushed a commit where I replaced some usages of `string` with `Columns` (just in order to document better)